js/BonusWheel.js

```diff
@@ -2,25 +2,10 @@ class BonusWheel extends Wheel {
   constructor() {
     super();
     this.bonusPlayer = null;
-    this.spinValues = this.filterOutNotNumbers(this.generateSpinValues(500));
```

Comment:
Haha I guess 500 was a little much as a multiplier. Also made it hard to fit into the grandtotal boxes